tests/integration/sites_test.py

```diff
@@ -116,70 +106,80 @@ def test_atom_locations(self, vasp_sites):
                        rel_tol=1e-4)
 
     def test_atom_locations_parts(self, vasp_sites):
-        assert len(vasp_sites.atom_locations_parts) == self.n_parts
-        assert isclose(vasp_sites.atom_locations_parts[0]['48h'],
+        assert len(vasp_sites.atom_locations_parts()) == self.n_parts
+        assert isclose(vasp_sites.atom_locations_parts()[0]['48h'],
                        0.755111,
                        rel_tol=1e-4)
-        assert isclose(vasp_sites.atom_locations_parts[9]['48h'],
+        assert isclose(vasp_sites.atom_locations_parts()[9]['48h'],
                        0.738444,
                        rel_tol=1e-4)
 
-    def test_n_jumps(self, vasp_sites):
-        assert vasp_sites.n_solo_jumps == 1922
-        assert vasp_sites.n_jumps == 450
-        assert isclose(vasp_sites.solo_fraction, 4.2711, rel_tol=1e-4)
+    def test_n_jumps(self, vasp_jumps):
+        assert vasp_jumps.n_solo_jumps == 450
+        assert vasp_jumps.n_jumps == 462
```

Comment:
Why did this change by so much?

Reply:
Because the old value made no sense (discussed with Alexandros)